create-html-reports.sh

```diff
@@ -53,31 +53,31 @@ fi
 if [ "$ALL_REPORTS" != "" ]; then
     for report in $ALL_REPORTS; do
         report_name=$(basename -s .txt $report)
-        html=${TESTING_RESULTS_DIR}/${report_name}.html
+        html_report_abs_path=$(dirname $report)/${report_name}.html
```

Comment:
I believe `html` as assigned before `html=${TESTING_RESULTS_DIR}/${report_name}.html` is already a full path unless the user changes the `RESULTS_DIR` or the `RESULTS_ARCHIVE_DIR ` or the `OUTPUT_DIR`(not just changing the username). The changes you added makes `html_report_abs_path` a full path independent of the variable mentioned above. Was that the intention?

Reply:
yes, I was thinking rather than hardcode in `$TESTING_RESULTS_DIR` as was done before, that it should be independent of either tests or benchmarks, and this way it will just generate a .html file wherever it finds the report.

Reply:
I added a comment to hopefully better explain.